CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [3.1.0] - 2017-11-07
+### Added
+- Added `-f, --format` parameter to be able to specify a custom message format
```

Comment:
Would it make sense to instead provide a template of it? I am not sure how complex things might become with the formatting. Might me nice to specify an erb, jinja, etc style template and then just pass it the location of the template.

Reply:
Sounds like a good idea, will come up with a solution later today.